classes/04class/exercises/c04-iac02/starkmatt/vpc.tf

```diff
@@ -0,0 +1,103 @@
+data "aws_availability_zones" "iac-azs" {
+    state = "available"
+}
+
+resource "aws_vpc" "devopsacademy-iac" {
+  cidr_block = "${var.cidr_vpc}"
```

Comment:
since terraform 0.12 when interpolating single variables, you don't need to "${}" anymore
it will actually print a warning if you do that way
please update to: `cidr_block = var.cidr_vpc`

Reply:
can you please fix all entries across your code? be in mind that when you have a mix of string and variable values, then you can use "${}

i.e: Name = "${var.project_name}-vpc"